blueocean-core-js/src/js/components/DisablePipelineButton.jsx

```diff
@@ -15,7 +15,7 @@ export class DisablePipelineButton extends Component {
 
         this.state = {
             disabled: this.props.pipeline.disabled,
-            submitingChange: false,
+            submittingChange: false,
```

Comment:
This might be considered an API change, in which case we probably shouldn't make it and should instead document the api botch.

Reply:
@jsoref  Could you move this change to a different PR.  The documentation and comment fixes and are safe and easy.  I don't know enough to be sure on this change yet.

Reply:
moved to #2069